app/src/main/res/layout/tag_item.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView
```

Comment:
Can we create some common style for language items and tags items?

Reply:
Ok, I will do it